guide/how-it-works/silent-payments.md

```diff
@@ -0,0 +1,360 @@
+
+</div>
+
+This scheme improves label-related features since any labels added to a static address get auto-applied to all derived on-chain addresses (detectable only by the receiver), eliminating the friction in manually adding labels to addresses or transactions. This not only makes payment tracking possible, but also useful during automatic or manual coin selection.
+
+For eg: a contractor Alice can add labels to her silent payment address while sharing them with a certain client, her social media and her website. When she receives payments from any of these sources, the respective label will be detected by the wallet application in the on-chain address to infer the source of the payment as this client.
+
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/silent-payments/how-silent-payment-labels-work.png"
+   retina = "/assets/images/guide/how-it-works/silent-payments/how-silent-payment-labels-work@2x.png"
+   mobile = "/assets/images/guide/how-it-works/silent-payments/how-silent-payment-labels-work.png"
+   mobileRetina = "/assets/images/guide/how-it-works/silent-payments/how-silent-payment-labels-work@2x.png"
+   alt-text = "An image that illustrates how labels feature in silent payments works."
+   caption = "."
+   width = 1024
+   height = 801
+   layout = "full-width"
+%}
+
+
+These labelling and detection features can also be useful for:
+
+- Invoices: merchant software like BTCPay can use labels to match incoming payments with invoices. This is required because on-chain addresses themselves cannot be predicted on used to make the mapping
+- Static deposit addresses for exchanges: Users looking to deposit bitcoin at exchanges may get confused when they see changing/different deposit addresses. Exchanges can now share static addresses with users; this will keep deposit addresses the same while allowing the exchange to map deposit transactions to users
+
+
+### Contacts
+
+Contacts are another feature that are greatly improved with BIP-352. Since users can safely use the same silent payment address for multiple payments, it is natural for them to want to store these for future use. Contacts are a great way to store them in terms that users can intuit: names and faces/images. The Contacts page here provides good guidance about the topic.
+
+Labels help Contacts in another interesting way by enabling ‘sender contacts’ where the user can create Contacts for parties they only receive bitcoin from. This is accomplished by adding a different label for every contact.
+
+{% include image-gallery.html pages = page.images_contacts %}
+
+Overall, Silent payments present a great opportunity to start basing payments UX in terms of people instead of addresses, and applications.
+
+{% include tip/recommendation.html %}
+
+Wallets should create an in-app contact list and avoid storing bitcoin payment information in the operating system’s default contact list since it cannot guarantee its safety for data breaches or phone hacks.
```

Comment:
I don't think it is even possible to add bitcoin payment information to OS contact lists. AFAIC, those are pretty locked down.

Reply:
It is possible to add custom labels for email, phone fields in iOS and store info there (see screenshot below) but that's fair coz I don't know any apps that already do something like this. Would you suggest I delete this?

![kjbvkjdf](https://github.com/user-attachments/assets/baf475de-4145-4202-9d4f-535426e5f6ae)

Reply:
Regardless, this copy has been removed in a commit a few weeks back...